### PR TITLE
feat(calling): Call user indicators

### LINKED ISSUE #1595: feat(calling): show when participant mutes or records
Blink either does or will very soon emit an event when the participant state changes. The participant state will be updated to indicate if they are recording. Uplink will be able to show who is muted, deafened, or recording.

### PR DESCRIPTION
**Pull request by Flemmli97**
### What this PR does 📖

- Adds indicators to the active call ui that shows if a user is muted, deafened or recording
~~- ATM it seems that the values in the call are not updated correctly~~

### Which issue(s) this PR fixes 🔨

- Resolve #1595

### Note

Needs #1609 first to fully function

### CODE REVIEW

**Comment by phillsatellite** [comment]
Very very very minor, it looks like the edge of the profile picture cuts through the muted icon and looks a little weird 

Screenshot 2023-12-15 at 11 30 36 AM

File: ui/src/components/media/calling.rs [hunk 1]
```diff
@@ -710,13 +711,45 @@ pub fn CallUserImageGroup(cx: Scope<CallUserImageProps>) -> Element {
...
     cx.render(rsx!(
-        visible.iter().map(|(speaking, user)| {
+        visible.iter().map(|(speaking, user_state, user)| {
             rsx!(div {
                 class: format_args!("call-user {}", if *speaking {"speaking"} else {""}),
                 UserImage {
                     platform: user.platform,
                     image: user.photo.clone(),
                 }
+                user_state.as_ref().map(|s|{
```
Comment (lgmarchi) [inline ui/src/components/media/calling.rs]: We are repeating this code block in line 777, could we do a function and don't repeat code?

**Comment by Flemmli97** [comment]
> Very very very minor, it looks like the edge of the profile picture cuts through the muted icon and looks a little weird
> 
> Screenshot 2023-12-15 at 11 30 36 AM

increase icon background so it should be more clear

**Comment by phillsatellite** [comment]
@Flemmli97 so I retested with the latest commits and still seems to look a little off. Like the middle part of the microphone is missing

Screenshot 2023-12-18 at 3 15 48 PM
Screenshot 2023-12-18 at 3 14 53 PM

**Comment by Flemmli97** [comment]
> @Flemmli97 so I retested with the latest commits and still seems to look a little off. Like the middle part of the microphone is missing
> 
> Screenshot 2023-12-18 at 3 15 48 PM Screenshot 2023-12-18 at 3 14 53 PM

thats intendeted. the icon just looks that way

**Comment by Flemmli97** [comment]
> @Flemmli97 so I retested with the latest commits and still seems to look a little off. Like the middle part of the microphone is missing
> 
> Screenshot 2023-12-18 at 3 15 48 PM Screenshot 2023-12-18 at 3 14 53 PM

thats intended. the icon just looks like that. its the same icon as the mute button when you click on it

**Review by phillsatellite — APPROVED** [review]

**Review by sdwoodbury — APPROVED** [review]

**Pull request merged**